src/rest/AllowedValuesPatternType.ts

```diff
@@ -1,13 +1,13 @@
 /**
- * Values that are allowed for the [allowedValuesPatternType]{@link IGroupByRequest.allowedValuesPatternType} parameter.
+ * The allowed values for the [allowedValuesPatternType]{@link IGroupByRequest.allowedValuesPatternType} parameter.
```

Comment:
Suggestion:

The possible values for the [`allowedValuesPatternType`]{@link IGroupByRequest.allowedValuesPatternType} property of the `IGroupByRequest` interface.